index.html

```diff
@@ -9,11 +9,15 @@
     <script src="app.js" defer></script>
     <title>inNotion UI Library | Home</title>
   </head>
-  <body>
-    <main class="centered-text">
+  <body class="content">
+    <header class="row-flex p-2 header sticky-header box-shd">
+      <h1 class="h3 title wt-text">inNotion UI Library</h1>
```

Comment:
is the h3 class needed here? Just asking as I got confused after seeing an h3 class on an h1 tag.

Reply:
Since one heading was required per page, I've added this in the h1 tag, but the size was the issue, so gave the `h3` class.. shall I change it if it's not proper?

Reply:
Oooh! It makes sense but it's a little confusing, that's all :sweat_smile: